packages/platform-sdk-dot/src/services/message.ts

```diff
@@ -24,7 +24,7 @@ export class MessageService implements Contracts.MessageService {
 
 	public async sign(input: Contracts.MessageInput): Promise<Contracts.SignedMessage> {
 		try {
-			const keypair = this.#keyring.addFromUri(input.mnemonic);
+			const keypair = this.#keyring.addFromUri(input.mnemonic!);
```

Comment:
This needs a proper check that the mnemonic is set now that it isn't enforced by TypeScript.